### DISCUSSION TITLE
Are you planning to improve

### DISCUSSION THREAD

**Discussion by pi314267** [ME-MarvinE/XCalendar#124] [Q&A] [answered]
Hello, first of all, thank you for providing an amazing calendar function. You seem to be a very talented developer. I have been using Xcalendar and noticed that the Event Calendar is logging graphics-related logs such as "[OpenGLRenderer] Davey! duration=769ms;". Are you planning to improve the speed performance of the application?

**Comment by ME-MarvinE** [comment, marked as answer]
Is this in Forms or Maui? Maui is several times slower with same code so don't know how to progress there. I've only identified that the main cause is bindings, even just a single Label with a binded value will slow it down drastically.

For both, I have done as much as I can without the knowledge of platform specific APIs.

I'll always want to improve it but I don't know where to look anymore. If suggestions come up then of course I'll try them.

**Reply by pi314267** [reply]
Yes, it's in MAUI. I understand the performance issue you're experiencing. I will try some possible solutions such as optimizing the bindings, and I will let you know if I find anything that could help.

**Reply by migueBarrera** [reply]
@pi314267 Any update? Did you improve performance?

**Reply by ME-MarvinE** [reply]
@migueBarrera Issue #112 might be something to look at if you haven't seen it already.
